config/application.rb

```diff
@@ -4,31 +4,51 @@
 # Pick the frameworks you want:
 require 'active_model/railtie'
 require 'active_job/railtie'
-require 'active_storage/engine'
+# require 'active_record/railtie'
```

Comment:
Is it worth just removing thee lines instead of commenting them out? We can always retrieve from GitHub if needs be?